### ISSUE TITLE
Create separate pipeline for Hawaii

### ISSUE THREAD

**Issue by andrew-rist-tw** [PublicDataWorks/complaint-manager#230]
~Blocked by #229~

### Business Context:
We need to be able to deploy a Hawaii image (rather than an NOIPM image) to the CI and Staging environment and we also need to be able to make a deployment to Hawaii's production accounts once we have deskchecked and tested the Hawaii changes in the lower environments

**As a** Hawaii instance-files image
**I want** to be included in a complaint-manager docker image and deployed to various environments
**So that** I can be tested in lower environments and ultimately used in higher environments

### In Scope
1. Add a separate pipeline that loads Hawaii instance-files and runs all the same tests with them
2. Put manual approval steps in front of all New Orleans lower environment deploy steps so that the Hawaii image is automatically deployed but New Orleans can be deployed over it when selected
3. Add a branch in the release pipeline with its own manual deploy step for Hawaii so that when cutting a release branch you can choose to deploy to Hawaii, New Orleans or both

### Out of Scope
1.
2.
3.

### Open Questions
1.
2.
3.

## Acceptance Criteria

1. Scenario: 
Given
When
Then

2. Scenario: 
Given
When
Then

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)
0. Discuss the technical implementation of this card.
1. Discuss how this feature would be included in or affect the Audit Log.
2. In respect to security, what could go wrong? Brainstorm threats using STRIDE model, especially if this card introduces a new technical architecture or design pattern. (TIMEBOX: 7 mins) https://docs.google.com/document/d/18AFl1d3SRieI4d_8R3GyEazqQOycLNDHGN_vOYIHuiE/edit#heading=h.aio9zb5j63wl
3. Ensure any open questions have been resolved with stakeholders.
4. Determine if documentation should be referenced or created.
5. Ensure to include all necessary mockups to this card.
6. Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
7. Discuss why this feature would not be implemented behind a toggle / what would happen if this card was partially done while a release branch was being cut.
8. If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous  cards in the sequence have been finished.
9. Have we added edge cases as ACs that we're considered from a QA perspective
10. Does this card overlap with any entities outlined in [API Housekeeping](https://www.pivotaltracker.com/story/show/173963545)?  

**Notes:**
